app/policies/feed_policy.rb

```diff
@@ -1,14 +1,18 @@
 class FeedPolicy < ApplicationPolicy
+  def show?
+    PodcastPolicy.new(token, resource.podcast).show?
+  end
+
   def create?
-    token&.authorized?(account_id)
+    PodcastPolicy.new(token, resource.podcast).create?
```

Comment:
Hrm... following the logic in Publish - you don't need to be able to Create/Delete the podcast to CRUD the feeds within the podcast.  So I think create/destroy should still be chained to `update?`.

But the `show?` looks correct to me here.